metricbeat/module/kubernetes/event/event.go

```diff
@@ -87,6 +88,7 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		dedotConfig:   dedotConfig,
 		watcher:       watcher,
 		watchOptions:  watchOptions,
+		skipOlder:     true,
```

Comment:
This value should come from the config like `skipOlder: config.SkipOlder`.
Now it is hardcoded to true.

Reply:
🤦🏼‍♂️  you are right! Thanks for catching this!